--- a/Makefile	2022-08-23 16:00:36.904272406 -0700
+++ b/Makefile	2022-08-23 16:08:56.327628320 -0700
@@ -6,7 +6,8 @@
 
 BINARY_PATH = ${DESTDIR}${PREFIX}/bin
 MAN_PATH = ${DESTDIR}${MANPREFIX}/man1
-SRC_PATH = ${DESTDIR}${SRCPREFIX}/bsp-layout
+SRC_PREL = ${SRCPREFIX}/bsp-layout
+SRC_PATH = ${DESTDIR}${SRC_PREL}
 
 MANPAGE = ${MAN_PATH}/bsp-layout.1
 
@@ -21,7 +22,7 @@
 	cp src/layout.sh layout.sh.tmp
 	bash update_version.sh ${SRC_PATH}
 	cp -f ${SRC_PATH}/layout.sh layout.sh.tmp
-	sed "s|{{SOURCE_PATH}}|${SRC_PATH}|g" layout.sh.tmp > ${SRC_PATH}/layout.sh # Update source path
+	sed "s|{{SOURCE_PATH}}|${SRC_PREL}|g" layout.sh.tmp > ${SRC_PATH}/layout.sh # Update source path
 	rm layout.sh.tmp
 	bash update_version.sh ${MANPAGE}
 	chmod 644 ${MANPAGE} # Manpage permission
