--- a/Makefile	2022-11-24 17:38:36.752649557 -0800
+++ b/Makefile	2022-11-24 17:41:34.575000962 -0800
@@ -65,7 +65,6 @@
 	mkdir -p $(DESTDIR)$(PREFIX)/share/doc/keyd/
 	mkdir -p $(DESTDIR)$(PREFIX)/share/doc/keyd/examples/
 
-	-groupadd keyd
 	install -m755 bin/* $(DESTDIR)$(PREFIX)/bin/
 	install -m644 docs/*.md $(DESTDIR)$(PREFIX)/share/doc/keyd/
 	install -m644 examples/* $(DESTDIR)$(PREFIX)/share/doc/keyd/examples/
